WG-CC/transitiontoolkit-backend
# feat: implement make with requirements.txt

# Description
I installed a `Makefile` which has commands to run what developers need to run the files we are using right now. I would like someone on Mac to make sure the commands work on Mac and that they can run the `Makefile` commands as I tested them on Windows

# Closes issue(s)

Resolves #2 

# How to test
To test you want to go to my branch and try to run the following commands in this order

1.  `make venv` : This should install the venv on your local computer
2.  `make install`: This should install the requirements.txt
3.  `make run`: This should run the endpoint for `/` and you should see `Hello World`
4.  `make clean`: At the moment doesn't do anything noticeable
5.  `make freeze`: Should update the requirements.txt file if you add or change dependencies. Though at the moment for testing it won't change any as you aren't adding any

**THIS NEEDS TO BE TESTED BY SOMEONE ON A MAC TO MAKE SURE THE COMMAND `MAKE` IS WORKING PROPERLY ON THAT OS. I CODE ON A WINDOWS**

# Changes include
FastApi was added as a dependency. 

# Checklist
